tiltak/src/test/kotlin/no/nav/amt/tiltak/deltaker/repositories/BrukerRepositoryTest.kt

```diff
@@ -56,22 +58,10 @@ class BrukerRepositoryTest : FunSpec({
 	}
 
 	test("oppdaterVeileder should update veileder") {
-		val fodselsnummer = "12345678910"
-		val fornavn = "Per"
-		val mellomnavn = null
-		val etternavn = "Testersen"
-		val telefonnummer = "74635462"
-		val epost = "per.testersen@test.no"
-		val ansvarligVeilederId = UUID.fromString("4118216f-b46d-44a1-90c5-d0732e861d6e")
-		val nyAnsvarligVeilederId = UUID.fromString("5e8790a9-7339-4ea2-ae75-54aac33f6c4d")
-		val brukerInsert = BrukerInsertDbo(fodselsnummer, fornavn, mellomnavn, etternavn, telefonnummer, epost, ansvarligVeilederId, null)
-
-		repository.insert(brukerInsert)
-
-		repository.oppdaterVeileder(fodselsnummer, nyAnsvarligVeilederId)
+		repository.get(BRUKER_1.fodselsnummer)?.ansvarligVeilederId shouldBe NAV_ANSATT_1.id
```

Comment:
Her mener jeg at man burde kalle oppdater veileder to ganger, ikke anta at testdataene oppfører seg på en spesiell måte. Evt burde brukeren være navngitt for å vise at det er en spesiell type bruker